docs/indepth/installation.rst

```diff
@@ -173,49 +175,49 @@ Using Amazon Web Services
 
 #. Add the *Key* to your local key chain (download it from the AWS web interface first if necessary):
 
-   .. code-block:: sh
+   .. prompt:: bash laptop$
 
-      yourlaptop$ mv ~/Downloads/my-instance.pem ~/.ssh/
-      yourlaptop$ chmod 600 ~/.ssh/my-instance.pem
-      yourlaptop$ ssh-add ~/.ssh/my-instance.pem
+      mv ~/Downloads/my-instance.pem ~/.ssh/
+      chmod 600 ~/.ssh/my-instance.pem
+      ssh-add ~/.ssh/my-instance.pem
 
-#. Look up the public DNS name or public IP address of the new instance and log in as user "fedora", e.g.:
+#. Look up the public DNS name or public IP address of the new instance and, depending on the OS, log in as user "fedora", "centos", or "ubuntu" e.g.:
 
-   .. code-block:: sh
+   .. prompt:: bash laptop$
 
-      yourlaptop$ ssh fedora@ec2-AA-BB-CC-DD.eu-west-1.compute.amazonaws.com
+      ssh fedora@ec2-AA-BB-CC-DD.eu-west-1.compute.amazonaws.com
 
-#. Allow SSH access for the ``root`` user
+#. Allow SSH access for the ``root`` user, then log out.
 
    .. task:: install_ssh_key
-      :prompt: [fedora@aws]#
+      :prompt: [aws]$
 
-   You should now be able to log in as "root" and the ``authorized_keys`` file should look approximately like this:
+#. Log back into the instances as user "root", e.g.:
 
-   .. code-block:: sh
+   .. prompt:: bash laptop$
 
-      yourlaptop$ ssh root@ec2-54-72-149-156.eu-west-1.compute.amazonaws.com
-      [root@aws]# cat /root/.ssh/authorized_keys
-      ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCe6FJDenfTF23azfJ2OVaorp3AsRQzdDlgkx/j0LrvQVyh95yMKL1GwVKuk8mlMGUEQiKImU6++CzTPu5zB2fpX+P5NrRZyBrokwp2JMQQD8lOqvvF7hw5bq2+8D8pYz11HkfEt9m5CVhLc1lt57WYnAujeRgaUhy9gql6r9ZI5aE8a3dpzxjP6S22er1/1dfLbecQaVM3cqpZVA6oAm8I6kJFyjiK6roRpaB2GTXTdpeGGiyYh8ATgDfyZPkWhKfpEGF5xJtsKSS+kFrHNqfqzDiVFv6R3fVS3WhdrC/ClqI941GeIM7PoDm3+KWlnaHJrjBX1N6OEBS8iEsj+24D username
+      ssh root@ec2-AA-BB-CC-DD.eu-west-1.compute.amazonaws.com
 
-#. Log back into the instances as user "root", e.g.:
+#. Upgrade the Kernel
 
-   .. code-block:: sh
+   Some operating systems require an updated kernel.
 
-      yourlaptop$ ssh rootec2-AA-BB-CC-DD.eu-west-1.compute.amazonaws.com
+   Fedora kernels older than ``3.16.4`` have a bug that affects Flocker's use of ZFS.
+   On Fedora, run:
 
-#. Upgrade the Kernel
+   .. task:: upgrade_kernel fedora-20
+      :prompt: [root@fedora]#
 
-   Kernels older than ``3.16.4`` have a bug that affects Flocker's use of ZFS.
+   On CentOS, run:
 
-   .. task:: upgrade_kernel
-      :prompt: [root@aws]#
+   .. task:: upgrade_kernel centos-7
+      :prompt: [root@centos]#
```

Comment:
This being `[root@centos]` and the next one (same machine, same user) being `[root@aws]` is weird.